Cloud Application Manager/Tutorials/ebcli-tutorial.md

```diff
@@ -0,0 +1,442 @@
+* Access to Cloud Application Manager [Site](https://cam.ctl.io/#/login) or to an on-premises CAM appliance.
+* The user must have an existing account to login into using username and password or his preferred authentication mechanism.
+
+
+## Installation of ebcli command line tool
+
+Before you start using ebcli, you need to install it using pip which is a package manager for Python packages, or modules if you like.
+
+If you don't have pip installed. Please follow this guide to install it into your computer: https://pip.pypa.io/en/stable/installing/
+
+**Terminal Command**
+
+Open a terminal in your computer and execute the installation command:
+
+```
+pip install ebcli
+```
+
+When the installation process ends pip should have installed ebcli and all of its required dependencies.
+
+
+### Login into your Cloud Application Manager account with ebcli
+
+Before you can control your Cloud Application Manager account with ebcli, you need to connect your account in Cloud Application Manager with ebcli.
+
+The ebcli needs a token to login into your Cloud Application Manager account.
+
+**Steps to create a token**
+
+* Open https://cam.ctl.io if you are using SaaS or https://YOUR_APPLIANCE_FQDN if you are using on premises of cam
+* Login with your username and password or your preferred authentication mechanism
+* Click on your user name menu (at top right corner of CAM user interface)
+* Click on 'Authentication Tokens'
+* Write a descriptive name for your token, i.e. ebcli
+* Click on 'Create Token'
+* Click on clipboard buttom at right of your token to copy it
+
+**Terminal Command**
+
+Open a terminal in your computer and execute the login command:
+
+```
+ebcli login
+```
+
+You will be prompt for introduccing your 'Authentication Token:', just paste your token, and hit Enter.
+
+Now your computer will ask your keyring password to store your Cloud Application Manager token safely in your keyring.
+
+### Show available ebcli commands
+
+**Terminal Command**
+
+Open a terminal in your computer and execute the login command:
+
+```
+ebcli
+```
+
+ebcli will show usage help like this one:
+
+```
+usage: ebcli [-h] [--url URL] [--token TOKEN] [--debug] [--verbose] [-j]
+             {boxes,build,delete,deploy,get,export,export-instance,import,instances,login,logout,set,poweron,reconfigure,reinstall,shutdown,terminate,workspaces}
+             ...
+ebcli: error: too few arguments
+```
+
+**General Parameters**
+
+| Get Boxes Option | Description |
+|-------------------|-------------|
+| --url | By default it's https://cam.ctl.io but you can use this to specify your APPLIANCE FQDN. |
+| –-token | You can use it to specify your token in every command if you don't want to store it into your keyring. |
+| –-debug | Show debug information to provide detailed traces to support in case of issues. |
+| –-verbose | Show detailed information of operations and api calls requested to Cloud Operation Manager. |
+
+
+### Box commands using ebcli
+
+## Get boxes
```

Comment:
```suggestion
#### Get boxes
```
Do not use level 2 titles (the minimum to use in a KB is level 3, and since in this case is a section under boxes, you should use level 4 instead.